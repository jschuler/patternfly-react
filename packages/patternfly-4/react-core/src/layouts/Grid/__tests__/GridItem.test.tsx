// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { GridItem } from '../GridItem';

describe('GridItem test', () => {
  it('GridItem should match snapshot', () => {
    const view = shallow(
      <GridItem
        children={<>ReactNode</>}
				className={"''"}
				span={null}
				rowSpan={null}
				offset={null}
				sm={undefined}
				smRowSpan={undefined}
				smOffset={undefined}
				md={undefined}
				mdRowSpan={undefined}
				mdOffset={undefined}
				lg={undefined}
				lgRowSpan={undefined}
				lgOffset={undefined}
				xl={undefined}
				xlRowSpan={undefined}
				xlOffset={undefined}
				xl2={undefined}
				xl2RowSpan={undefined}
				xl2Offset={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
