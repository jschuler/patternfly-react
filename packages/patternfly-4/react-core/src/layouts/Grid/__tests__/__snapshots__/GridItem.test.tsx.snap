// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItem test GridItem should match snapshot 1`] = `
<div
  className="pf-l-grid__item ''"
>
  ReactNode
</div>
`;
